variables.tf

```diff
@@ -1,4 +1,187 @@
-variable "example" {
-  description = "Example variable"
-  default     = "hello world"
+variable "number_of_broker_nodes" {
+  type    = number
+  description = "The desired total number of broker nodes in the kafka cluster. It must be a multiple of the number of specified client subnets. It must be a multiple of the number of specified client subnets."
 }
+
+variable "kafka_version" {
+  type    = string
+  description = "Specify the desired Kafka software version"
+}
+
+variable "broker_instance_type" {
+  type    = string
+  description = "Specify the instance type to use for the kafka brokers"
+}
+
+variable "broker_volume_size" {
+  type    = number
+  default = 1000
+  description = "The size in GiB of the EBS volume for the data drive on each broker node"
+}
+
+variable "vpc_id" {
+  type = string
+  description = "VPC ID where subnets will be created (e.g. `vpc-aceb2723`)"
+}
+
+variable "subnet_ids" {
+  type = list(string)
+  description = "Subnet IDs"
+}
+
+variable "enabled" {
+  type        = bool
+  default     = true
+  description = "Set to false to prevent the module from creating any resources"
+}
+
+variable "namespace" {
+  type        = string
+  default     = ""
+  description = "Namespace, which could be your organization name or abbreviation, e.g. 'eg' or 'cp'"
+}
+
+variable "name" {
+  type        = string
+  default     = ""
+  description = "Solution name, e.g. 'app' or 'cluster'"
+}
+
+variable "zone_id" {
+  type = string
+  description = "Route53 DNS Zone ID"
+}
+
+variable "stage" {
+  type        = string
+  default     = ""
+  description = "Stage, e.g. 'prod', 'staging', 'dev', OR 'source', 'build', 'test', 'deploy', 'release'"
+}
+
+variable "environment" {
+  type        = string
+  default     = ""
+  description = "Environment, e.g. 'prod', 'staging', 'dev', 'pre-prod', 'UAT'"
+}
+
+variable "delimiter" {
+  type        = string
+  default     = "-"
+  description = "Delimiter to be used between `namespace`, `environment`, `stage`, `name` and `attributes`"
+}
+
+variable "attributes" {
+  type        = list(string)
+  default     = []
+  description = "Additional attributes (e.g. `1`)"
+}
+
+variable "label_order" {
+  type        = list(string)
+  default     = []
+  description = "The naming order of the id output and Name tag"
+}
+
+variable "tags" {
+  type        = map(string)
+  default     = {}
+  description = "Additional tags (e.g. `map('BusinessUnit','XYZ')`"
+}
```

Comment:
Move standard variables to the top in standard order

Reply:
Done